pyproject.toml

```
@@ -29,7 +29,7 @@ include = [
 [tool.poetry.dependencies]
 python = ">=3.9.2,<3.13"
 # Used for local development
-nautobot = ">=2.4.2,<3.0.0"
+nautobot = "^2.4.16"
```

Trying to keep these compliant with PEP508

```suggestion
nautobot = ">=2.4.16,<3.0.0"
```